templates/shrine/src/shrine-simple-item.html

```diff
@@ -20,9 +20,12 @@
         @apply(--layout-horizontal);
         @apply(--layout-center-center);
         @apply(--layout-wrap);
+        position: relative;
         text-align: center;
         width: 100%;
         height: 100%;
+        /* create a layer to only invalidate this layer for :hover */
+        transform: translateZ(0);
```

Comment:
Should this move to `:host::after` since that's the one actually display the box-shadow on hover?